front/components/dialogs/ResendAuthEmail.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <default-dialog
+    :value="value"
+    :button-ok-disabled="sendingEmail || blockSendAgainTime > 0"
+    button-cancel-text="Back To Log in"
+    :button-ok-text="sendAgainText"
+    header="Didn't get a confirmation email?"
+    @input="closeDialog"
+    @cancel="backToLogIn"
+    @ok="sendEmail"
+  >
+    <div class="emailSentMessage">
+      <span v-if="sendingEmail"
+        >Sending another email to <strong>{{ email }}</strong
+        >...</span
+      >
+      <span v-else>
+        We've sent another login email to you at <strong>{{ email }}</strong
+        >. Make sure the email address is spelled correctly, and try checking
+        your spam or junk folder. Click <strong>Back to Log In</strong> and
+        input correct email if you have mistaken.
+      </span>
+    </div>
+  </default-dialog>
+</template>
+
+<script>
+import * as Sentry from '@sentry/browser'
```

Comment:
we should do this. its importing from browser which could fail on ssr. And also nuxt sentry provide this.$sentry which should be used instead

Reply:
Ok, replaced with this.$sentry